File: Python/4. Bucles/Ejercicio #2.py
'''
2. Hacer un programa que realice la serie de fibonacci -> 1 1 2 3 5 8 13
'''

N = int(input("Numero de elementos de la secuencia: "))

x = 0  
y = 1
z = 1
i = 0

while i<N:    
    if i != N-1:
        print(f"{z}", end=",")
    else:
        print(f"{z}", end=".")
    
    z = x + y;

    x = y;
    y = z;
    
    i=i+1

File: 1. C++/5. Arreglos o Vectores/Ejercicio #2.cpp
/*2. Escribe un programa que defina un vector de numero y calcule la multiplicacion acumulada de sus elementos.*/

#include<iostream>

using namespace std;

int main()
{
    int numeros[] = {1,2,3,4,5}, PRO = 1;

    for (int i = 0; i < 5; i++)
    {
        cout<<numeros[i];
        PRO *= numeros[i]; 
        if (i < 4)
        {
            cout<<" + ";
        }
    }
    cout<<" = "<<PRO<<endl;
    return 0;
}